app/Http/Controllers/PayloadHistoryController.php

```diff
@@ -3,30 +3,41 @@
 namespace App\Http\Controllers;
 
 use Illuminate\Http\Request;
+use App\Enums\PayloadHistoryStatus;
 use App\Repositories\Interfaces\PayloadHistoryRepositoryInterface as PayloadHistoryRepository;
 use App\Repositories\Interfaces\MessageHistoryRepositoryInterface as MessageHistoryRepository;
-
+use App\Repositories\Interfaces\WebhookRepositoryInterface as WebhookRepository;
+use Auth;
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Namespaces.UseDeclaration.SpaceAfterLastUse</br>**Severity**: error</br>**File**: app/Http/Controllers/PayloadHistoryController.php</br>**Line**: 10</br>There must be one blank line after the last USE statement; 0 found;